src/utils/isGitRepository.js

```diff
@@ -0,0 +1,13 @@
+// @flow
```

Comment:
We're in the middle of slowly moving away from Flow to TypeScript. So new files should definitely be written in TS.

Reply:
Thanks. I wasn't sure if I could just go ahead and write it in Typescript. 
And yes, it seems Carlos suggests removing the helper 😄